query-node/schema.graphql

```diff
@@ -269,37 +322,40 @@ type Video @entity {
   "Whether the Video is supposed to be publically displayed"
   isPublic: Boolean!
 
-  "Video curation status set by the Curator"
-  isCurated: Boolean!
+  "Flag signaling whether a video is censored."
+  isCensored: Boolean!
 
   "Whether the Video contains explicit material."
   isExplicit: Boolean!
 
   license: LicenseEntity!
 
+  "Encoding of the video media object"
+  encoding: VideoMediaEncoding!
```

Comment:
VideoMediaEncoding is not correct, its the old content directory schema entity. In new design it comes from the protobuf metadata. And I don't think we will have 'pre-populated' encoding entities in the query-node database. Or should we?

Reply:
VideoMediaEncoding entity has been removed and relevant properties from protobuf have been integrated into Video entity.